BEIMA.DB.Schemas/device-schema.txt

```diff
@@ -28,10 +28,8 @@
             "fileName": String
         }
     ],
-    photos: [
-        {
-            "fileUid": String,
-            "fileName": String
-        }
-    ]
+    photos: {
```

Comment:
I thought you renamed this to "photo" in the other PR?

Reply:
Thought I did too, just updated it